xlators/cluster/afr/src/afr-common.c

```diff
@@ -5151,6 +5154,33 @@ afr_lk_cbk(call_frame_t *frame, void *cookie, xlator_t *this, int32_t op_ret,
     return 0;
 }
 
+int
+afr_lk_txn(void *opaque)
+{
+    call_frame_t *frame = NULL;
+    afr_local_t *local = NULL;
+    int child_index = -1;
+    afr_private_t *priv = NULL;
+    fd_t *fd = NULL;
+    xlator_t *this = NULL;
+
+    frame = (call_frame_t *)opaque;
+    local = frame->local;
+    child_index = local->child_index;
+    this = frame->this;
+    priv = this->private;
+    fd = local->fd;
+
+    afr_fix_open(fd, this, true);
+
+    STACK_WIND_COOKIE(frame, afr_lk_cbk, (void *)(long)child_index,
```

Comment:
nit: unsure if we need STACK_WIND_COOKIE here, since we have child_index stored in local->child_index now.
Requires some minor changes elsewhere.

Reply:
Sorry, I don't know what needs to be modified. Please tell me, Thank you!

Reply:
@mykaul  Sorry, Ask you for advice. Now I don‘t know how to modify it. Please tell me.
Thank you!